src/actions/MastodonModelAction.C

```diff
@@ -58,8 +62,13 @@ MastodonModelAction::MastodonModelAction(const InputParameters & params)
     _disp_variables({"disp_x", "disp_y", "disp_z"}),
     _vel_auxvariables({"vel_x", "vel_y", "vel_z"}),
     _accel_auxvariables({"accel_x", "accel_y", "accel_z"}),
+    _dim(getParam<unsigned int>("dim")),
```

Comment:
This would get changed to `<MooseEnum>`, but the use of `_dim` remains the same.